src/blog/branching-out-phase-2-of-corda-flavored-ganache.md

```diff
@@ -1,6 +1,6 @@
 ![Branching out to Corda-flavored Ganache](/img/blog/branching-out-phase-2-of-corda-flavored-ganache/chameleon-corda-truffle-blog-header.png)
 
-April has been a big month for Truffle Suite! Earlier this month we branched out and announced Tezos integration into the Truffle command line tool. If you remember, back in December we released a beta [Corda-flavored Ganache](https://www.trufflesuite.com/blog/unwrap-the-corda-flavored-ganache-beta) as part of our initial support for the Corda platform. Today, we are excited to share [Phase 2 of Corda-flavored Ganache, v2.4.0](https://github.com/trufflesuite/ganache/releases) with you! Phase 2 includes more robust features for Corda including Corda CRaSH Shell, network map visualization, and a version picker.
+April has been a big month for Truffle Suite! Earlier this month we branched out and announced Tezos integration into the Truffle command line tool. If you remember, back in December we released a beta [Corda-flavored Ganache](https://www.trufflesuite.com/blog/unwrap-the-corda-flavored-ganache-beta) as part of our initial support for the Corda platform. Today, we are excited to share [Phase 2 of Corda-flavored Ganache, v2.4.0](https://github.com/trufflesuite/ganache-ui/releases) with you! Phase 2 includes more robust features for Corda including Corda CRaSH Shell, network map visualization, and a version picker.
```

Comment:
Not sure how much this matters but should this link go to a specific release?

Reply:
Corda is in all stable releases now, so it is probably okay as is.